src/startupscriptgenerator/php/main.go

```diff
@@ -0,0 +1,32 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package main
+
+import (
+	"flag"
+	"startupscriptgenerator/common"
+)
+
+func main() {
+	common.PrintVersionInfo()
+
+	appPathPtr := flag.String("appPath", ".", "The path to the application folder, e.g. '/home/site/wwwroot/'.")
+	startupCmdPtr := flag.String("startupCommand", "apache2-foreground", "Command that will be executed to start the application server up.")
+	outputPathPtr := flag.String("output", "run.sh", "Path to the script to be generated.")
```

Comment:
we should also accept a command from the user, if provided

Reply:
other than startupCmdPtr?